src/Helper/Participant.php

```diff
@@ -408,6 +414,30 @@ private function parseRdrParticipant($participant)
         if (isset($participant->aian)) {
             $this->isAIAN = $participant->aian;
         }
+
+        if (isset($participant->nphWithdrawal)) {
+            $this->nphWithdrawal = $participant->nphWithdrawal;
```

Comment:
What is the intention behind setting these properties? Usually, we set properties of the Participant class when there is a computation involved with the RDR participant object. But here since you are just saving the property directly not sure what's the purpose behind it.

Reply:
I believe PHPStan throws that error specifically when attempting to access the property in the Participant class. Are you experiencing any PHPStan errors when you remove those setters? Considering that you are only setting those properties in the Participant class and not accessing them, I think it's safe to remove them.